ui.frontend/clientlib.config.js

```diff
@@ -52,42 +52,52 @@ module.exports = {
         js: {
           cwd: 'clientlib-dependencies',
           files: ['**/*.js'],
-          flatten: false
+          flatten: false,
         },
         css: {
           cwd: 'clientlib-dependencies',
           files: ['**/*.css'],
-          flatten: false
-        }
-      }
+          flatten: false,
+        },
+      },
     },
     {
       ...libsBaseConfig,
       name: 'clientlib-base',
       categories: ['saas-aem-module.base'],
       dependencies: ['saas-aem-module.dependencies'],
       assets: {
-        // Copy entrypoint scripts and stylesheets into the respective ClientLib
+        // Copy entrypoint scripts into the respective ClientLib
         // directories
         js: {
           cwd: 'clientlib-base',
           files: ['**/*.js'],
-          flatten: false
+          flatten: false,
         },
+      },
+    },
+    {
+      ...libsBaseConfig,
+      name: 'wknd-sample',
+      categories: ['saas-aem-module.wknd.sample'],
+      dependencies: ['saas-aem-module.dependencies'],
+      assets: {
+        // Copy entrypoint stylesheets into the respective ClientLib
+        // directories
         css: {
           cwd: 'clientlib-base',
           files: ['**/*.css'],
-          flatten: false
+          flatten: false,
         },
 
         // Copy all other files into the `resources` ClientLib directory
         resources: {
```

Comment:
@nhirrle I moved the resources (fonts, images...) with the styling instead of the javascript. Hope that makes sense.
Can the resources be accessible from another clientlib? Should be right?

Eg. I want to access an image in clientlib-base (JS) from WKND-sample (CSS).

Reply:
as discussed we should ideally use class names in the js /  dome instead of referencing resources directly in javascript.